tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -212,31 +212,51 @@ public async Task ValidateCcsHeadersForInteractiveAuthCodeFlowAsync()
 
             Assert.IsNotNull(authResult);
             Assert.IsNotNull(authResult.AccessToken);
+        }
+
+        [TestMethod]
+        public async Task Interactive_Arlington_MultiCloudSupport_AADAsync()
```

Comment:
Integration tests are expensive, especially these ones with UI. Best to execute them only for .NET Classic or for .NET Core (by default they execute on both). See comment on line 61. I'd add this test in the `#if` block.